### PR TITLE
OCPBUGS 6843: Adding details about docker build strategy

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.9+

Issue:
https://issues.redhat.com/browse/OCPBUGS-6843

Link to docs preview:
https://55633--docspreview.netlify.app/openshift-enterprise/latest/applications/creating_applications/creating-applications-using-cli.html#build-strategy-detection_creating-applications-using-cli

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@ingvagabund @zhouying7780 Can you please review? I need confirmation on whether it is actually correct that it looks for a Dockerfile to detect for docker (I pulled that info from v3 docs somewhere, so I'm not confident that it's correct).

Preview: https://55633--docspreview.netlify.app/openshift-enterprise/latest/applications/creating_applications/creating-applications-using-cli.html#build-strategy-detection_creating-applications-using-cli

**Comment by ingvagabund** [comment]
@coreydaley PTAL

**Comment by coreydaley** [comment]
Looks good to me

File: modules/applications-create-using-cli-source-code.adoc [hunk 1]
```diff
@@ -5,8 +5,9 @@ With the `new-app` command you can create applications from source code in a loc
 
 The `new-app` command creates a build configuration, which itself creates a new application image from your source code. The `new-app` command typically also creates a `Deployment` object to deploy the new image, and a service to provide load-balanced access to the deployment running your image.
 
-{product-title} automatically detects whether the pipeline or source build strategy should be used, and in the case of source builds, detects an appropriate language builder image.
+{product-title} automatically detects whether the pipeline, source, or docker build strategy should be used, and in the case of source builds, detects an appropriate language builder image.
```
Comment (zhouying7780) [inline modules/applications-create-using-cli-source-code.adoc:8]: better to change to : "and in the case of source build"

**Pull request merged**